### ISSUE TITLE
[BUG]: The placeholders are not replaced for the method getEnvironmentPublicKey

### ISSUE THREAD

**Issue by bogdanmanate-kion** [octokit/rest.js#437]
### What happened?

Calling the getEnvironmentPublicKey should return the public key for a specific environment, instead a 404 error code is returned. 
and it can be seen from the error stack that the placeholders for owner and repo are not replaced.
And example can be found here: https://runkit.com/bogdanmanate-kion/666985abdb0e230008ad9ce7

### Versions

Octokitjs 20.0.2, Node v18.11.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gr2m** [comment]
> `repositories/octokit/restjs`

this should be `/repos/octokit/rest.js`, see https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/e96924537da2ff115643efebfb6b58e740a35e38/src/generated/endpoints.ts#L129-L131

**Comment by bogdanmanate-kion** [comment]
@gr2m Yes, that is the endpoint, "GET /repos/{owner}/{repo}/environments/{environment_name}/secrets/public-key" from what I see in the error log, only the environment_name is replaced. Please have a look at the provided runkit.

**Comment by mpern** [comment]
ran into the same issue while using actions/github-script.

I cannot get the environment public key:

```
RequestError [HttpError]: Not Found
    at .../_actions/actions/github-script/v7/dist/index.js:9537:21
... {
  status: 404,
  response: {
    url: 'https://<github enterprise>/api/v3/repositories//environments/foo/secrets/public-key?owner=bar&repo=baz',
```

`environment_name` is replaced in the URL, and interestingly enough the GET parameters `owner` and `repo` are also correct. But owner/repo is missing in the path
